src/components/itemcontextmenu.js

```diff
@@ -346,11 +346,7 @@ define(["apphost", "globalize", "connectionManager", "itemHelper", "appRouter",
                     break;
                 case "copy-stream":
                     var downloadHref = apiClient.getItemDownloadUrl(itemId);
-                    navigator.clipboard.writeText(downloadHref).then(function () {
-                        require(["toast"], function (toast) {
-                            toast(globalize.translate("CopyStreamURLSuccess"));
-                        });
-                    }, function () {
+                    var textareaCopy = function () {
```

Comment:
```suggestion
                    var textAreaCopy = function () {
```

Use proper lowerCamelCase please.

Otherwise, this looks okay.